src/main/java/org/apache/sysds/runtime/instructions/SPInstructionParser.java

```diff
@@ -39,55 +39,10 @@
 import org.apache.sysds.lops.WeightedUnaryMMR;
 import org.apache.sysds.runtime.DMLRuntimeException;
 import org.apache.sysds.runtime.instructions.cp.CPOperand;
-import org.apache.sysds.runtime.instructions.spark.AggregateTernarySPInstruction;
-import org.apache.sysds.runtime.instructions.spark.AggregateUnarySPInstruction;
-import org.apache.sysds.runtime.instructions.spark.AppendGAlignedSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.AppendGSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.AppendMSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.AppendRSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.BinUaggChainSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.BinarySPInstruction;
-import org.apache.sysds.runtime.instructions.spark.BuiltinNarySPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CSVReblockSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CastSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CentralMomentSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CheckpointSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CompressionSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CovarianceSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CpmmSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CtableSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CumulativeAggregateSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.CumulativeOffsetSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.DnnSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.IndexingSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.MapmmChainSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.MapmmSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.MatrixReshapeSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.MultiReturnParameterizedBuiltinSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.PMapmmSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.ParameterizedBuiltinSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.PmmSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.QuantilePickSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.QuantileSortSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.QuaternarySPInstruction;
-import org.apache.sysds.runtime.instructions.spark.RandSPInstruction;
-import org.apache.sysds.runtime.instructions.spark.ReblockSPInstruction;
```

Comment:
A change for imports can be done in a separate PR with `[MINOR]` tag. Otherwise this would impede the PR progress.

So, for this PR only these _deletions_ be reverted.